image/tarexport/save.go

```diff
@@ -10,6 +10,8 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/containerd/platforms"
+
 	"github.com/containerd/containerd/images"
```

Comment:
🙈 stray empty line

Reply:
Ugh, thanks! I really need to sort out my Goland settings 🙈